src/Resources/views/Admin/UiElement/button.html.twig

```diff
@@ -4,9 +4,11 @@
     element fields :
         label
         link
+        target
         align
 #}
 {% set align = element.align is defined and element.align is not empty ? element.align : 'center' %}
+{% set target = element.target is defined and element.target is not empty ? element.target : '_self' %}
 <p style="text-align: {{align}};">
-    <a href="{{ element.link }}" class="ui primary button massive">{{ element.label }}</a>
+    <a href="{{ element.link }}" target="{{ element.target }}" rel="noopener noreferrer" class="ui primary button massive">{{ element.label }}</a>
```

Comment:
The `rel="noopener noreferrer"` must be added only if target is blank

You can compare if blank with the constant defined before with Twig `constant` method

Reply:
I've submitted another commit with a potential fix. let me know what you think.

Reply:
nah, I still have to tinker around a bit to make it work. will do this tomorrow.